package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "AsyncAPI website",
   "scripts": {
-    "dev": "node scripts/build-post-list.js && node scripts/build-roadmap.js && next -p 8080",
+    "dev": "node scripts/build-post-list.js && node scripts/build-roadmap.js && next -p 5000",
```

Comment:
why changing port for dev?

Reply:
Ooops I forgot to revert it. I'm using port 8080, that's why I changed it to 5000. I'll revert it back on my next PR

Reply:
what you mean about next PR? why next PR? I noticed you created few PRs already. Why not just push to the branch and get this PR updated?

Reply:
Sorry I meant updated PR 🙂